### DISCUSSION TITLE
Unable to load the map within the territory of mainland China,and mandatory requirements to Please select a school from the map

### DISCUSSION THREAD

**Discussion by ghost** [community/community#116898] [GitHub Education] [answered]
### Hi everyone,

Unable to load the map within the territory of mainland China,and mandatory requirements to Unable to load the map within the territory of mainland China.
What can I do?
Thank you for your help.

**Comment by nick-cjyx9** [comment, marked as answer]
The requests to bing.com will be auto-redirected to cn.bing.com in China,and this breaks the display of bing map.You can apply proxy to these domains:
`bing.com`,`www.bing.com`,`cn.bing.com`

Hope can help you

**Reply by ghost** [reply]
Thanks.

**Reply by ghost** [reply]
哇！同胞！
